### PR TITLE
add None return for peak_times in calculate_feature() of extraFELFeature class

### PR DESCRIPTION
**Pull request by darshanmandge**
To return None when the peak_times is None in calculate_feature function of extraFELFeature.

### CODE REVIEW

File: bluepyopt/ephys/efeatures.py [hunk 1]
```diff
@@ -426,6 +426,12 @@ def calculate_feature(
         from .extra_features_utils import calculate_features
 
         """Calculate feature value"""
+        if peak_times is None:
+            if return_waveforms:
+                return None, None
+            else:
+                return None
+
```
Comment (AurelienJaquier) [inline bluepyopt/ephys/efeatures.py:434]: ci says there are blank spaces here. could you remove them?

Comment (darshanmandge) [reply bluepyopt/ephys/efeatures.py]: Done. Thanks.

**Comment by AurelienJaquier** [comment]
@alejoe91 I'm pinging you to let you know we are making this change related to LFPy efeature

**Comment by alejoe91** [comment]
Thanks! Sounds fair :)

**Review by AurelienJaquier — APPROVED** [review]

**Pull request merged**